test/iterators.jl

```diff
@@ -149,6 +149,29 @@ for xs in Any["abc", [1, 2, 3]]
     @test take(drop(take(xs, 3), 1), 1) === take(drop(xs, 1), 1)
 end
 
+# takewhile
```

Comment:
This looks redundant with the name of the testset. Same below.

Reply:
done